### PR TITLE
rename gcf -> cf everywhere

### PR DESCRIPTION
**Pull request by drnic**

### CODE REVIEW

**Comment by jberkhahn** [comment]
Hey there,

For the sake of cleanliness of our commit history, we prefer to have PR's squashed into a single commit. Would you mind squashing the 3 commits into a single one?

**Comment by drnic** [comment]
done

**Comment by julz** [comment]
lgtm, we might have to also update the version of cf-test-helpers in Cartridge.lock -- I think the version we have in Cartridge.lock still expects 'gcf' to exist (in fact, cats/cf-test-helpers still expected 'gcf' to exist until [about a month ago](https://github.com/cloudfoundry-incubator/cf-test-helpers/commit/eaf5edb6ce3c155cb9f3993089933a97177b4810)).

File: README.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ application, see the instructions on "Running PAT as a Cloud Foundry App" below.
...
       *(Ignore any warnings about "no buildable Go source files")
-      *(Ignore errors in src/github.com/cloudfoundry-incubator/pat/workloads/gcf.go")
+      *(Ignore errors in src/github.com/cloudfoundry-incubator/pat/workloads/cf.go")
     cd $GOPATH/src/github.com/cloudfoundry-incubator/pat
     gocart
 
 5) See [CF CLI] (https://github.com/cloudfoundry/cli) for instructions on installing `cf`
 
-Note: It is important that `cf` is accessable on your `$PATH`, and that it has either been symlinked to `gcf` or renamed to `gcf`.
+Note: It is important that `cf` is accessable on your `$PATH`, and that it has either been symlinked to `cf` or renamed to `cf`.
```
Comment (julz) [inline README.md:53]: This sentence doesn't make much sense now :-)

Comment (drnic) [inline README.md:53]: This PR would be faster if someone merged it and fixed the other bits :)

Comment (julz) [inline README.md:53]: fine :)

Comment (drnic) [inline README.md:53]: xoxo

**Pull request merged**